source/_integrations/utility_meter.markdown

```diff
@@ -23,9 +24,36 @@ Some utility providers have different tariffs according to time/resource availab
 Sensors created with this integration are persistent, so values are retained across restarts of Home Assistant. The first cycle for each sensor will be incomplete; a sensor tracking daily usage will start to be accurate the next day after the integration was activated. A sensor tracking monthly usage will present accurate data starting the first of the next month after being added to Home Assistant.
 </div>
 
-## Configuration
-
-To enable the Utility Meter Sensor in your installation, add the following to your `configuration.yaml` file:
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+Name:
+  description: The name the sensor should have. It can be changed again later.
+Input sensor:
+  description: The sensor entity providing utility readings (energy, water, gas, heating).
+Meter reset cycle:
+  description: How often to reset the counter.
+Meter reset offset:
+  description: >
+    Cycle reset occur at the beginning of the period. This option enables
+    the offsetting of these beginnings.
```

Comment:
We should mention config flows only supports an offset counted in days, and that yaml gives more freedom.
We should also mention that if the combination of "meter reset cycle" and "meter reset offset" is too limiting, yaml supports CRON patterns.